app/Http/Controllers/Api/V1/Integration/SmithsonianIOAAPIClientController.php

```diff
@@ -84,8 +84,8 @@ public function getContentDetail(Request $request)
      *
      * Get a list of search filter data w.r.t filter category
      *
-     * @bodyParam the term category or search filter name. Required String. Only Allowed values:culture, data_source, date, object_type, online_media_type, place, topic, unit_code
-     * @bodyParam starts_with the optional string prefix filter. Example: Any alphabet or string
+     * @bodyParam category required string The term category or search filter name. Only Allowed values:culture, data_source, date, object_type, online_media_type, place, topic, unit_code
+     * @bodyParam starts_with optional string Prefix filter. Example: Any alphabet or string
```

Comment:
"Example" should be values not description.

Reply:
Done!